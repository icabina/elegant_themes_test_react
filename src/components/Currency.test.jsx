import { convert } from "./currency";
import "whatwg-fetch";
import { rest } from "msw";
import { setupServer } from "msw/node";

it("converts correctly", async () => {
  const rate = await convert("USD", "CAD");
  expect(rate).toEqual(1.42);
});
