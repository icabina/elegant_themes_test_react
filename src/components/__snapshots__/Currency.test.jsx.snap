// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contacts: Array["Bob", "Marcy"] 1`] = `
Array [
  "Bob",
  "Marcy",
]
`;
